nautobot/dcim/models/racks.py

```diff
@@ -363,8 +365,9 @@ def get_reserved_units(self):
                 reserved_units[u] = r
         return reserved_units
 
-    def get_elevation_svg(
+    def get_elevation(
```

Comment:
Renaming this is arguably a breaking API change - can you add back in a `get_elevation_csv` method that has the same args/kwargs and wraps this "new" function?